### PR TITLE
Fixing CI Docker Image

### PR DESCRIPTION
**Pull request by tristal**
Seems like the current image doesn't exist anymore.  Updating to the correct version resolves the issue.

### CODE REVIEW

File: docker-compose.env.production.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: "2"
```
Comment (spboyer) [inline docker-compose.env.production.yml]: This file is not needed for the sample, was this added by mistake?

File: service-a/public/index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 <body style="margin-left:10px; margin-right:10px;">
     <div ng-controller="MainController">
-        <h2>Server Says Hello</h2>
+        <h2>Server Says Hi</h2>
```
Comment (spboyer) [inline service-a/public/index.html]: Why the change here? Seems unnecessary.

**Review by spboyer — CHANGES_REQUESTED** [review]
Change to the base image is good, the other changes are not needed for the sample. Can you revert those?

**Comment by tristal** [comment]
Sorry, @spboyer.  Incorrectly pushed a few local commits to this one.  Reverted and fixed to only be the image change now.
